railties/lib/rails/generators/rails/app/templates/config/spring.rb

```diff
@@ -1,6 +1,6 @@
-%w[
-  .ruby-version
-  .rbenv-vars
-  tmp/restart.txt
-  tmp/caching-dev.txt
-].each { |path| Spring.watch(path) }
+Spring.watch(
+  ".ruby-version",
+  ".rbenv-vars",
+  "tmp/restart.txt",
+  "tmp/caching-dev.txt"
+)
```

Comment:
I'm not super familiar with spring's internals but judging by https://github.com/rails/spring/blob/156dcbf134c990917adce799409d6695bea21285/lib/spring/watcher/abstract.rb#L65-L70 this wouldn't introduce any process spawning differences, right?